src/inputs/Textarea.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div class="textarea">
```

Comment:
I've added BEM css classes to all components, please change this to `form__textarea`.